EvotoApi/Areas/Management/Connections/RegistrarConnection.cs

```diff
@@ -120,5 +120,33 @@ private static IRestRequest CreateRequest(string uri, Method method, object data
             exception.Data["content"] = res.Content;
             throw exception;
         }
+
+        public static async Task<IList<SingleRegiSettingResponse>> ListRegistrarSettings()
+        {
+            var req = CreateRequest("/settings/list", Method.GET);
+            var res = await MakeApiRequest(req);
+
+            if (res.StatusCode == HttpStatusCode.OK)
+                return JsonConvert.DeserializeObject<IList<SingleRegiSettingResponse>>(res.Content);
+
+            var exception = new Exception("Error listing registrar settings");
```

Comment:
I feel like this should be a custom exception if you want to add metadata. Is the metadata needed or just for debugging purposes?